event_pubsub/alembic/versions/c43c9dd9a19f_remove_logindex_for_unique_constraint.py

```diff
@@ -0,0 +1,34 @@
+"""remove logindex for unique constraint
+
+Revision ID: c43c9dd9a19f
+Revises: 632dd0534d07
+Create Date: 2022-04-19 22:00:57.017561
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'c43c9dd9a19f'
+down_revision = '632dd0534d07'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint('uq_converter_agix_ev', 'converter_agix_events_raw', type_='unique')
+    op.create_unique_constraint('uq_converter_agix_ev', 'converter_agix_events_raw', ['transactionHash'])
```

Comment:
Only at the time of reprocessing the events, we should have this constraint otherwise transaction hash and log index should be set as unique parameters.

Reply:
At any point in time, we will have a unique transaction if we reprocess or for any events. Only staking we would have the same transaction hash with a different log index. Since the converter is responsible for only listening to burn and mint, we are removing the unique constraint of `transaction_hash + log_index` to `transaction_hash`